python/ray/utils.py

```diff
@@ -232,6 +232,18 @@ def hex_to_binary(hex_identifier):
     return binascii.unhexlify(hex_identifier)
 
 
+def compute_job_id_from_driver(driver_id):
+    assert isinstance(driver_id, ray.WorkerID)
+    return ray.JobID(driver_id.binary()[0:ray.JobID.size()])
+
+
+def compute_driver_id_from_job(job_id):
+    assert isinstance(job_id, ray.JobID)
```

Comment:
This should wrap the appropriate C++ method (using Cython) to make sure everything is consistent

Reply:
This is a temporary helper function to covert ID types. And it'll be removed once we separate the `WorkerID` from `UniqueID`. Using cython is too heavy for a temporary helper function I think. :)

Reply:
I'm ok with having a temporary helper function like this, but then there should be a comment explaining that and a TODO to remove it.

Reply:
done